nntrainer/tensor/var_grad.cpp

```diff
@@ -19,10 +19,8 @@
 namespace nntrainer {
 
 Var_Grad::Var_Grad(const TensorDim &dim, const Tensor::Initializer init,
-                   bool ng, bool alloc_now_, const std::string &name) :
-  dim(dim),
-  need_gradient(ng),
-  alloc_now(alloc_now_) {
+                   bool need_gradient, bool alloc_now,
+                   const std::string &name) {
```

Comment:
Isn't there any svace issues if we remove this initialization?

Reply:
the corresponding member variables have been removed from the class, so it's ok.